modules/local/get_test_data.nf

```diff
@@ -0,0 +1,18 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options       = [:]
+def options          = initOptions(params.options)
+
+process GET_TEST_DATA {
+    container "docker.io/yuukiiwa/git:latest"
+
+    output:    
+    path "test-datasets/fast5/$barcoded"  , emit: ch_input_path
+
+    script:
+    barcoded = workflow.profile.contains('test_bc_nodx') ? "nonbarcoded" : "barcoded"
+    """
+    git clone https://github.com/nf-core/test-datasets.git --branch nanoseq --single-branch
```

Comment:
What is the purpose of this module? Why not give direct raw github links in the test data input samplesheet?

Reply:
This is for staging [all test fast5 files](https://github.com/nf-core/test-datasets/tree/nanoseq/fast5/barcoded) from `--input_path [fast5 directory]`

Reply:
Ah, input directory - I see. OK!

For nf-core/eager we just cloned the test-data repository as a GitHub Actions CI step, but I guess nice to bundle it with the pipeline itself for others to use :+1: